account_debitnote/models/account.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2019 Ecosoft Co., Ltd (http://ecosoft.co.th/)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, _
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    debitnote_sequence_id = fields.Many2one(
+        comodel_name='ir.sequence',
+        string='Debit Note Entry Sequence',
+        help="""This field contains the information related to the numbering of
+                the debit note entries of this journal.""",
+        copy=False
+    )
+    debitnote_sequence_number_next = fields.Integer(
+        string='Debit Notes: Next Number',
+        help='The next sequence number will be used for the next debit note.',
+        compute='_compute_debitnote_seq_number_next',
+        inverse='_inverse_debitnote_seq_number_next'
+    )
+    debitnote_sequence = fields.Boolean(
+        string='Dedicated Debit Note Sequence',
+        help="""Check this box if you don't want to share the same sequence
+                for invoices and debit notes made from this journal""",
+        default=False
```

Comment:
```suggestion
        default=False,
```
boolean default false. you need to add this?